src/platforms/android/common/proguard.mdx

```diff
@@ -51,6 +51,13 @@ defaults.project=your-project
 defaults.org=your-org
 auth.token=YOUR_AUTH_TOKEN
 ```
+
+<Note>
+
+Values inside `sentry.properties` file should not be wrapped around quotes; you should only replace the placeholders. Otherwise, it may lead to authentication or resource errors.
+
+</Note>
```

Comment:
@mathrocco what made you believe it should have been wrapped in quotes? the example above isnt.

Reply:
I'm not used to this `.properties` file format, so in troubleshooting I accidentally left the quotes, which led me to unauthorized errors. Since it took me some time to realize it, I thought it could help save others' time.